src/vocutouts/workers.py

```diff
@@ -100,11 +100,16 @@ def cutout_range(
     repository = os.environ["CUTOUT_BUTLER_REPOSITORY"]
     collection = os.environ["CUTOUT_BUTLER_COLLECTION"]
     butler = Butler(repository, writeable=True, run=input_collection)
+
+    # This should only need to be done once, but it does have to be done once.
+    # For now, register the dataset type each time, but this is silly so we
+    # should come up with a better initialization process.  (Maybe do this as
+    # part of vo-cutouts init.)
     dataset_type = DatasetType(
         "cutout_positions",
         dimensions=data_id.keys(),
         universe=butler.registry.dimensions,
-        storage_class="AstropyQTable",
+        storageClass="AstropyQTable",
     )
     butler.registry.registerDatasetType(dataset_type)
```

Comment:
It's not completely crazy to do this each time since at least that will raise an exception if somehow your definition here doesn't match the definition already in registry. It returns a boolean indicating whether it was inserted or not. If you can guarantee it's fixed and set elsewhere then that will also be fine.